Remotion/Web/Development.Analyzers.IntegrationTests/Web.Development.Analyzers.IntegrationTests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
```

Comment:
Why do we need this?

Reply:
We need this to reference Remotion.Core.Context in the verifier.